src/pages/Encounters/tabs/EncounterNotesTab.tsx

```diff
@@ -666,6 +666,7 @@ export const EncounterNotesTab = ({ encounter }: EncounterTabProps) => {
                               }
                             }
                           }}
+                          className="flex-1 min-h-[40px] max-h-[200px]"
```

Comment:
let's not hardcode arbitrary pixel values when there are alternatives available!